test/test_metrics.jl

```diff
@@ -3,6 +3,10 @@ module TestMetrics
 using Phylo
 using Test
 
+if VERSION < v"1.5"
```

Comment:
Should add Compat to test's deps and `using Compat: contains` here.

Reply:
I did try this already, but it didn't work for me for some reason - see 2db1355

Reply:
Okay, I give up - I've tried really hard to get it to work, but there's something very weird going on I agree. However, I'm really not keen on this new definition of contains, so can we just change `contains(x, y)` to `occursin(y, x)` in the two places it's used...